packages/edit-site/src/hooks/use-theme-style-variations/use-theme-style-variations-by-property.js

```diff
@@ -51,14 +51,12 @@ export function removePropertyFromObject( object, property ) {
  * A convenience wrapper for `useThemeStyleVariationsByProperty()` that fetches the current theme style variations,
```

Comment:
Since `useThemeStyleVariationsByProperty` is deleted, this comment and the file names should probably be updated.